starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java

```diff
@@ -51,10 +57,13 @@ public List<Inventory> findAll() {
    * @return Created/Updated Inventory.
    */
   public Inventory create(Inventory inventory) {
-    // TODO
-    return null;
+    // TODO TASK 1 
+    inventory.setId(null);
+    mongoTemplate.save(inventory);
+    return inventory;
```

Comment:
Line 62 and line 63 can be combined into one line (i.e., return the mongo template function); view the return object of mongo template's `.save()` to see why: https://docs.spring.io/spring-data/mongodb/docs/current/api/org/springframework/data/mongodb/core/MongoTemplate.html#save(T).

Add the keyword `this` to `mongoTemplate.save(inventory)` for consistent syntax of using `mongoTemplate`.